### PR TITLE
[WIP] Remove jquery-ui from the Fees->'EDI History' module

### LINKED ISSUE #3005: Remove jquery-ui from the Fees->'EDI History' module
Remove jquery-ui from the Fees->'EDI History' module

### PR DESCRIPTION
**Pull request by im-Amitto**
Fixes #3005 

@bradymiller can i get a valid csv file for testing dialog.

below added by bradymiller:
Up For Grabs demo for this PR: https://www.open-emr.org/wiki/index.php/Development_Demo#Delta_-_Up_For_Grabs_Demo

### CODE REVIEW

**Comment by bradymiller** [comment]
@stephenwaite and @sjpadgett , I remember somebody discussing example files to test the EDI module. Where were those again?

**Comment by bradymiller** [comment]
Up For Grabs demo for this PR is at: https://www.open-emr.org/wiki/index.php/Development_Demo#Delta_-_Up_For_Grabs_Demo
(it resets daily around midnight pacific time and uses the code on this PR)

File: interface/billing/edih_view.php [hunk 1]
```diff
@@ -33,7 +33,7 @@
     <meta http-equiv="content-type" content="text/html;charset=utf-8" />
     
     <!-- TODO: Address no_bootstrap here !-->
-    <?php Header::setupHeader(['no_main-theme', 'no_bootstrap', 'datetime-picker', 'datatables', 'datatables-jqui', 'datatables-jqui-theme', 'datatables-scroller', 'datatables-scroller-jqui-theme', 'jquery-ui', 'jquery-ui-sunny']); ?>
+    <?php Header::setupHeader(['no_main-theme', 'datetime-picker', 'datatables', 'datatables-jqui', 'datatables-jqui-theme', 'datatables-scroller', 'datatables-scroller-jqui-theme', 'jquery-ui', 'jquery-ui-sunny']); ?>
```
Comment (bradymiller) [inline interface/billing/edih_view.php]: regarding the datatables-jqui and datatables-jqui-theme , this substitution worked well elsewhere:
https://github.com/openemr/openemr/pull/3048/files

for scroller, guessing may need to substitute the datatables-scroller-jqui-theme (datatables.net-scroller-jqui) with https://www.npmjs.com/package/datatables.net-scroller-bs4

**Comment by stephenwaite** [comment]
> @stephenwaite and @sjpadgett , I remember somebody discussing example files to test the EDI module. Where were those again?

hi, attached a sample 837 file which you can upload through fees->edihistory and then click submit and process, then click the csv tables tab and choose a csv table and all dates and click submit
[EDI_837_5010_Usecase1.txt](https://github.com/openemr/openemr/files/4273248/EDI_837_5010_Usecase1.txt)

File: interface/billing/edih_view.php (hunk 1 above)
Comment (im-Amitto) [reply interface/billing/edih_view.php]: Thanks!, i was just thinking what alternative i have.

**Comment by tywrenn** [comment]
@im-Amitto I suggest looking at my pull in which I attempted fixing EDI History, #2927. This fixed the navigation and stuff yet the scripts I didn't get to fix too much.

**Comment by tywrenn** [comment]
@im-Amitto Still working on this?

**Comment by im-Amitto** [comment]
@tywrenn I actually shifted my focus on flutter openemr flutter but I will try to fix it in within 48 hours if it doesn't work let's close it as I can't abandon the flutter repo for too long.

**Comment by im-Amitto** [comment]
Closing the PR as @stu01509  is planning to fix it up

**Pull request closed without merging**